packages/frontend/src/features/post/hooks/useVotes/useVotes.ts

```diff
@@ -36,17 +36,19 @@ export function useVotes() {
         }) => {
             const userState = await getGuaranteedUserState()
             let epochKeyProof
+            let onCancel = false
             if (votedNonce !== null && votedEpoch !== null) {
-                epochKeyProof = await userState.genEpochKeyProof({
+                epochKeyProof = await userState.genEpochKeyLiteProof({
                     nonce: votedNonce,
                     epoch: votedEpoch,
                 })
+                onCancel = true
             } else {
                 const nonce = getEpochKeyNonce(actionCount)
                 epochKeyProof = await userState.genEpochKeyProof({ nonce })
             }
```

Comment:
Why do we use different types of proof when voting and unvoting?

Reply:
Changed to standardise the use of genEpochKeyLiteProof.